### DISCUSSION TITLE
Video Tag in Next.js Throws 404 Error

### DISCUSSION THREAD

**Discussion by shkarDiary** [vercel/next.js#65057] [Help]
### Summary

I’m encountering a 404 error when using the <video> tag in my Next.js project. Here’s my code snippet:
<video>
          <source src="/public/video/game.mp4" type="video/mp4" />
          Your browser does not support the video tag.
</video>

The video file (mosh.mp4) is located in the public folder. When I load the page, the video doesn’t play, and the console shows a 404 error for the video file.

Any insights on how to resolve this issue would be greatly appreciated! Thank you. 

### Additional information

```js
<video>
          <source src="/public/video/game.mp4" type="video/mp4" />
          Your browser does not support the video tag.
        </video>
```

**Comment by nikos-chatzatoglou** [comment]
@shkarDiary please check if your are using the relative or the absolute path for rendering the video. You can understand this if you are using the "/" before the path (absolute) or you don't (relative). Also try to delete the change  `/public/video/game.mp4` to `/video/game.mp4`

**Comment by fareed13** [comment]
@shkarDiary this may be helpful
https://stackoverflow.com/questions/70185663/how-to-use-a-video-file-in-public-folder-in-next-js/79144026#79144026
